course_catalog/etl/ocw.py

```diff
@@ -20,6 +20,46 @@
 
 log = logging.getLogger()
 
+OCW_DEPARTMENT_CODE_TO_NAME = {
+    "1": "Civil and Environmental Engineering",
```

Comment:
I'm going to need the OCW department slug for an issue I'll be working on soon, so I'll probably be changing this to something like:

```python
OCW_DEPARTMENT_CODE_TO_NAME = {
   "1": {
             "name": "Civil and Environmental Engineering",
             "slug": "civil-and-environmental-engineering"
     },
     ....
}
```

But it's fine as is for this PR